### ISSUE TITLE
Invalid position given by 'W084'

### ISSUE THREAD

**Issue by divinesweet** [AtomLinter/linter-jshint#446]
JSHint returned a point that did not exist in the document being edited.
Rule: `W084`
Requested point: 300:47

Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0.3
Platform: win32
Current file's scopes: [
  "source.js",
  "comment.line.double-slash.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
Is your file indented with tabs?

**Comment by Arcanemagus** [comment]
Assuming that you are indenting your file with tabs this is another case of https://github.com/jshint/jshint/issues/3151. There are reports that if you set your `indent` property of your JSHint configuration to `1` this issue can be worked around. If you don't have a `.jshintrc` file the following should work:
```json
{
  "indent": 1
}
```

If anyone is directed here and _isn't_ using tabs to indent please provide code to reproduce this so an issue can be filed on JSHint to get this fixed there.
